CONTRIBUTING.rst

```diff
@@ -164,10 +173,12 @@ the Pull Request workflow as described above, but with the notebook title now ch
 the title itself (no "Draft:" or Baseline:"). The Pull Request is then reviewed by one of the project scientists, and merged when
 everyone is satisfied with the notebook.
 
-## Final Notebook - Maintenance Stage
+
+Final Notebook - Maintenance Stage
```

Comment:
I do not think that this step is happening anymore. @eteq is this true?

Reply:
Agreed! I may be wrong, but I think I argued against this even existing since I think no work like this can ever be "final" ;).

Maybe the point about the scientific context/final plot could be carried up to the "advanced" stage, though?  Right now this is the only part that says you revise from the "rough around the edges" version in baseline, so just adding that as part of "advanced" seems quite reasonable to me.